### PR TITLE
Yessenia Rodriguez javascript-light-bulb

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-02-02 at 12 26 49 PM
Screen Shot 2021-02-02 at 12 27 02 PM

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var clickCounter = 0;
+var $lightBulb = document.querySelector('.light-bulb');
+var $container = document.querySelector('.container');
+
+function lightOnAndOff(event) {
+  clickCounter++;
+  if (clickCounter % 2 !== 0) {
+    $lightBulb.className = 'buttonClick';
+    $container.className = 'lightOn';
+  } else if (clickCounter % 2 === 0) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:10]: This can just be an `else` statement.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good work so far! One note for you:

File: javascript-light-bulb/main.js (hunk 1 above)
Comment (yesseniarod) [reply javascript-light-bulb/main.js]: Screen Shot 2021-02-02 at 12 53 29 PM
Screen Shot 2021-02-02 at 12 53 14 PM

**Review by codingcodymiller — APPROVED** [review]
🎉

**Pull request merged**